client/src/main/scala/drt/client/actions/Actions.scala

```diff
@@ -85,6 +85,8 @@ object Actions {
 
   case class UpdateAirportInfos(infos: Map[String, AirportInfo]) extends Action
 
+  case class ToggleShowActual(state: Boolean) extends Action
```

Comment:
minor point - it's not actually a toggle if we pass it a boolean

Reply:
fair!